marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.module.scss

```diff
@@ -6,6 +6,12 @@
   margin-right: 2px;
 }
 
+.clearIcon{
+  float: right;
```

Comment:
It's best to avoid using float when possible since it removes elements out of the normal flow of the page.